test/integration/queue/consumers/ManagedConsumer-test.js

```diff
@@ -85,7 +87,11 @@ test('should be able to handle incoming message from a producer', async (t) => {
   sinon.assert.calledOnce(spy)
 
   const receivedMessage = spy.firstCall.args[0]
-  t.deepEqual(receivedMessage.clean(), testMessage.clean())
+  const receivedMessageCleaned = testMessage.clean()
```

Comment:
Should this be `receivedMessage.clean()`

Reply:
Yeah, the assertions below are going to always be true with the above assignment.